rest-api/dao/participant_summary_dao.py

```diff
@@ -279,7 +290,12 @@ def to_client_json(self, model):
       result['ageRange'] = get_bucketed_age(date_of_birth, clock.CLOCK.now())
     else:
       result['ageRange'] = UNSET
+    if 'organizationId' in result:
```

Comment:
Use format_json_organization, format_json_site here?

Reply:
If you were going to follow the pattern of format_json_site:

https://github.com/all-of-us/raw-data-repository/blob/8e897fb23127085dc3ff964ae7e70b75cf7e6ff8/rest-api/api_util.py#L87

then it would take care of setting "site" and deleting "siteId" on the object for you. That seems simpler.